lib/realtime/tenants/connect.ex

```diff
@@ -52,23 +52,23 @@ defmodule Realtime.Tenants.Connect do
   @doc """
   Returns the database connection for a tenant. If the tenant is not connected, it will attempt to connect to the tenant's database.
   """
-  @spec lookup_or_start_connection(binary(), keyword()) ::
+  @spec lookup_or_start_connection(binary(), binary(), keyword()) ::
           {:ok, pid()}
           | {:error, :tenant_database_unavailable}
           | {:error, :initializing}
           | {:error, :tenant_database_connection_initializing}
           | {:error, :rpc_error, term()}
-  def lookup_or_start_connection(tenant_id, opts \\ []) when is_binary(tenant_id) do
+  def lookup_or_start_connection(tenant_id, region, opts \\ []) when is_binary(tenant_id) do
```

Comment:
Should Connect.get_status take the whole tenant struct instead? 

Perhaps all these other things should not have to care about the region at all.

Reply:
it's more the tests that create the issues 🤔 and ideally we avoid passing the tenant as some of this stuff is called from RPC calls so it would mean sending a struct

Reply:
Looks like we already have a Nodes.get_region which is called via Nodes.get_node_for_tenant

Reply:
it's a `defp` but didn't see it honestly. will probably have it all under `Tenants` so it's bounded to the context of the tenant